File: backend/src/models/TipoUva.ts
export default class TipoUva {
  private nombre: string
  constructor(nombre: string) {
    this.nombre = nombre
  }

  getNombre(): string {
    return this.nombre
  }

  getTipoUva(): string {
    return this.getNombre()
  }

  setNombre(nombre: string) {
    this.nombre = nombre
  }
}

File: ReporteRankingVinos/src/main/java/com/Bonvino/ReporteRankingVinos/models/TipoUva.java
package com.Bonvino.ReporteRankingVinos.models;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
public class TipoUva {
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  private Integer id;
  private String nombre;

  public TipoUva(String nombre, String descripcion) {
    this.nombre = nombre;

  }

  public String getNombre() {
    return this.nombre;
  }
}
